bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/status/ConfigStatusService.java

```diff
@@ -107,14 +108,8 @@ private ConfigStatusInfo getConfigStatus(ConfigStatusProvider configStatusProvid
         for (ConfigStatusMessage configStatusMessage : configStatus) {
             String message = i18nProvider.getText(bundle, configStatusMessage.messageKey, null, locale,
                     configStatusMessage.arguments);
-            if (message == null) {
```

Comment:
I would prefer if we could keep the logging warning message. The i18n provider should only be invoked if the message key is not null